src/state/slices/portfolioSlice/portfolioSlice.ts

```diff
@@ -473,12 +473,9 @@ export const selectPortfolioCryptoBalanceByFilter = createSelector(
   selectAssetIdParamFromFilterOptional,
```

Comment:
please remove my comment above the name of this function, and change the usage in `AssetAccountRow` to use the `cryptoHumanBalance` now that this returns base units

Reply:
Done.